layer.yaml

```diff
@@ -15,6 +15,11 @@ options:
     puppet-version: '3'
     silent: True
 defines:
+  install_java:
+    type: string
+    default: 'openjdk-8-jdk'
```

Comment:
I was part of the "people" :) This config variable is fed into a bigtop config param that is marked as jdk. I know we have done some successful testing with jre but why risk it?